ipynb/examples/utils/testenv_example.ipynb

```diff
@@ -22,18 +38,86 @@
    "metadata": {
     "collapsed": false
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2016-12-08 11:54:02,743 INFO    : root         : Using LISA logging configuration:\n",
+      "2016-12-08 11:54:02,744 INFO    : root         :   /user_projects/lisa/lisa/lisa/logging.conf\n"
+     ]
+    }
+   ],
    "source": [
-    "import json\n",
-    "import time\n",
-    "import os"
+    "from conf import LisaLogging\n",
+    "LisaLogging.setup()\n",
+    "# For debug information use:\n",
+    "# LisaLogging.setup(level=logging.DEBUG)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test environment setup"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Test environment setup"
+    "**Target configuration**:\n",
```

Comment:
Would be good to specify that if the target configuration isn't specified then the construct defaults to the values in the target.conf file in the root dir of the LISA source tree.